README.md

```diff
@@ -8,13 +8,150 @@ It sends `commandStart` and `commandEnd` for execution of each the command.
 ## Version map
 |vscode-extension-telemetry-wrapper|vscode-extension-telemetry| 
 |---|---|
-|0.2.4|^0.0.18|
-|0.2.3 (Deprecated)|^0.0.17|
+|0.3.0 (next)|^0.0.18|
+|0.2.4 (latest)|^0.0.18|
+|~~0.2.3 (Deprecated)~~|~~^0.0.17~~|
 |0.2.2|^0.0.17|
 |0.2.1|0.0.10|
 |0.1.x|0.0.10|
 
-## Usage
+
+## Usage for 0.3.0 (New)
+### Examples
+- Initialize the wrapper on activation.
+    ```ts
+    import { initialize, instrumentOperation } from "vscode-extension-telemetry-wrapper";
+    export async function activate(context: vscode.ExtensionContext): Promise<void> {
+        // initialize the wrapper.
+        await initialize(extensionId, extensionVersion, aiKey);
+
+        // It instruments the activate operation to send related events.
+        await instrumentOperation("activation", doActivate)(context);
+    }
+
+    async function doActivate(_operationId: string, context: vscode.ExtensionContext): Promise<void> {
+        // Move your code here.
+    }
+    ```
+
+- Instrument your VS Code command.
+    ```ts
+    const name = "my.hello";
+    const myHello = (...args) => {
+        vscode.window.showInformationMessage("Hello: " + args.join(" "));
+    };
+    // _operationId contains a unique Id for each execution of `myHello`, in case you want to access it.
+    const instrumented = instrumentOperation(name, (_operationId, myargs) => myHello(myargs));
+    vscode.commands.registerCommand(name, instrumented);
+    ```
+
+- Mark an Error as user error.
```

Comment:
Also, have a section for `setErrorCode`.

Reply:
Updated.